python/cuspatial/cuspatial/tests/test_geodataframe.py

```diff
@@ -241,6 +241,20 @@ def test_post_slice(gpdf, post_slice):
     assert_eq_geo_df(gi[post_slice], cugpdf_back[post_slice])
 
 
+@pytest.mark.parametrize(
+    "inline_slice",
+    [slice(0, 12)]
+    + [slice(0, 10, 1)]
+    + [slice(0, 3, 1)]
+    + [slice(3, 6, 1)]
+    + [slice(6, 9, 1)],
```

Comment:
This seems like a rather strange way to construct a list? Did black change it? Why not just
```suggestion
    [slice(0, 12),
    slice(0, 10, 1),
    slice(0, 3, 1),
    slice(3, 6, 1),
    slice(6, 9, 1)],
```